encoding/rs/utils.go

```diff
@@ -20,15 +19,20 @@ func ToFrArray(data []byte) []fr.Element {
 		end := (i + 1) * uint64(encoding.BYTES_PER_COEFFICIENT)
 		if end >= uint64(len(data)) {
 			var padded [32]byte
```

Comment:
Looks like padded can just be a slice here instead of an array. Then you can drop the `[:]`